### PR TITLE
update swift version to 5.0

### PR DESCRIPTION
**Pull request by ostk0069**
I just update the project to swift version to `5.0`
I hope this would work 👍 

(please use xcode verison with `10.2` to check)

### CODE REVIEW

**Comment by tarunon** [comment]
Thanks contributing!!
But looks CI is failed, could you please try to fix CI's xcode version? 
https://github.com/tarunon/Instantiate/blob/master/.travis.yml#L5

File: Sources/InstantiateStandard/Extensions.swift [hunk 1]
```diff
@@ -26,6 +26,9 @@ extension IdentifierType {
 }
 
 #if os(iOS) || os(tvOS)
+
+    extension NSStoryboard.Name: IdentifierType {}
+    extension NSNib.Name: IdentifierType {}
```
Comment (tarunon) [inline Sources/InstantiateStandard/Extensions.swift]: `NSStoryboard` and `NSNib` is macOS classes. 🤔

Comment (ostk0069) [reply Sources/InstantiateStandard/Extensions.swift]: Oh, my bad. Thanks for letting me know.

**Comment by ostk0069** [comment]
Sorry for the many commit.
I found out that there are no `Apple TV 1080p` in this version. OS is too old for swift `5.0`

```
{ platform:tvOS Simulator, id:7343C157-7A73-4400-BF16-5474EB5EB388, OS:9.0, name:Apple TV 1080p }
{ platform:tvOS Simulator, id:514089CD-2DD8-4E2C-80FF-5FF2E665CB32, OS:9.1, name:Apple TV 1080p }
{ platform:tvOS Simulator, id:FE36E3AE-9796-4F29-B3F0-044689339A58, OS:9.2, name:Apple TV 1080p }
{ platform:tvOS Simulator, id:1DF2D08A-1F55-41C0-A3C6-43B6A215C967, OS:10.0, name:Apple TV 1080p }
{ platform:tvOS Simulator, id:413845E9-ECA5-4AB5-95CF-11566CF16C15, OS:10.1, name:Apple TV 1080p }
{ platform:tvOS Simulator, id:456EC0CF-C098-4EC1-890E-4DE0BD3C4527, OS:10.2, name:Apple TV 1080p }
```

Do you know any way I can fix for this?
For now, Test fails only with this reason, I guess.

**Comment by tarunon** [comment]
I think we can decide OS version in .travis.yml. 
https://docs.travis-ci.com/user/languages/objective-c/#device-destinations
Please check ↑ 🙏

**Comment by ostk0069** [comment]
My test has been passed.
I have small concern about this.
I don't think this code is not necessary anymore.
スクリーンショット 2019-05-08 10 13 41
Can I just delete it or not?

**Comment by tarunon** [comment]
Thanks fixing CI. Yeah I hope to remove all warnings, if you ok, please make new PR for fixing them. Thank you!

**Review by tarunon — APPROVED** [review]

**Pull request merged**